_dashboards/reporting-cli.md

```diff
@@ -0,0 +1,428 @@
+{: .note }
+
+### Example: Requesting a PNG report with environment variables set
+
+The following command requests a report with basic authentication in PNG format:
+
+```
+opensearch-reporting-cli --url https://localhost:5601/app/dashboards#/view/7adfa750-4c81-11e8-b3d7-01146121b73d --format png --auth basic --credentials admin:admin
+```
+
+Upon success, the report will download to the current directory.
+
+### Using Amazon SES to request an email with a report attachment
+
+To use Amazon SES as the email transport mechanism, the following prerequisites apply:
+
+- If a URL contains an exclamation point (!), then the history expansion needs to be disabled temporarily. The sender's email address must be verified by [Amazon SES](https://aws.amazon.com/ses/). The [AWS Command Line Interface (AWS CLI)](https://docs.aws.amazon.com/cli/latest/userguide/cli-chap-welcome.html) is required to interact with Amazon SES. To configure basic settings used by the AWS CLI, see [Quick configuration with `aws configure`](https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-quickstart.html#cli-configure-quickstart-config) in the AWS Command Line Interface user guide.
+- Amazon SES transport requires the `ses:SendRawEmail` role:
+
+```
+{
+  "Statement": [
+    {
+      "Effect": "Allow",
+      "Action": "ses:SendRawEmail",
+      "Resource": "*"
+    }
+  ]
+}
+```
+
+The following command requests an email with the report attached:
+
+```
+opensearch-reporting-cli --url https://localhost:5601/app/dashboards#/view/7adfa750-4c81-11e8-b3d7-01146121b73d --transport ses --from <sender_email_id> --to <recipient_email_id>
+```
+
+The following command uses default values for all other options. You can also set `OPENSEARCH_FROM`, `OPENSEARCH_TO`, and `OPENSEARCH_TRANSPORT` in your .env file and use the following command:
+
+```
+opensearch-reporting-cli --url https://localhost:5601/app/dashboards#/view/7adfa750-4c81-11e8-b3d7-01146121b73d
+```
+
+To modify the body of your email, you can edit the *index.hbs* file.
+
+### Example: Sending a report to an email address with SMTP
+
+To send a report to an email address with SMTP transport, you need to set the options `OPENSEARCH_SMTP_HOST`, `OPENSEARCH_SMTP_PORT`, `OPENSEARCH_SMTP_USER`, `OPENSEARCH_SMTP_PASSWORD`, and `OPENSEARCH_SMTP_SECURE` in your .env file.
+
+Once the transport options are set in your .env file, you can send the email using the following command:
+
+```
+opensearch-reporting-cli --url https://localhost:5601/app/dashboards#/view/7adfa750-4c81-11e8-b3d7-01146121b73d --transport smtp --from <sender_email_id> --to <recipient_email_id>
+```
+
+You can choose to set options using either your *.env* file or the command line argument values in any combination. Make sure to specify all required values to avoid errors.
+
+To modify the body of your email, you can edit the *index.hbs* file.
+
+## Environment variable limitations
```

Comment:
These are not env var limitation. maybe just Limitation is ok bcoz some are limitations due to dependencies and some due to system